apisix/plugins/proxy-rewrite.lua

```diff
@@ -163,6 +163,8 @@ local function check_set_headers(headers)
         if #field == 0 then
             return false, 'invalid field length in header'
         end
+
+        core.log.info("header field: ", field)
```

Comment:
Why not also show the value?

Reply:
Because the old version has this line, some test cases will check this output.

Reply:
OK, ignore it.